Assets/Scripts/UI/SwitchSelectMsg.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class SwitchSelectMsg : MonoBehaviour {
+    public GameObject textBox;
+    public GameObject switchMessage;
+    public GameObject acceptMessage;
+    private bool onlyOnce;
+    private bool onlyOnce2;
+    public Controller2 isghostMode;
+    // Use this for initialization
+    void Start()
+    {
+        //switchMessage = GameObject.FindGameObjectWithTag("SwitchMessage");
+        switchMessage.SetActive(false);
+        acceptMessage.SetActive(false);
+
+        onlyOnce = false;
+        onlyOnce2 = false;
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+
+        if (!textBox.gameObject.activeSelf && /*Input.GetKeyDown(KeyCode.X)*/ !isghostMode.ghostMode && !onlyOnce)
```

Comment:
not too sure what you're trying to do here, but if you're tryying to check whether textbox is enabled, check TextBoxManager.isActive, as i have an extra check on the state of player on that file